### DISCUSSION TITLE
Get the current node and content the user is typing on, cursor position.

### DISCUSSION THREAD

**Discussion by mandatoryGithubUser** [Milkdown/milkdown#1595] [Q&A]
How can I get the node/content at the position the user is currently typing. Their cursor position. 

For example, let's the content is like below, with the html version underneath. Right now the user is typing `Test3...`, but how can I only get that node / content? 

```
Test1
Test2
Test3...

<p>Test1</p>
<p>Test2</p>
<p>Test3...</p>
```

We're able to listen to current content, where it'd return ALL the values back, either in markdown or prose nodes. We can also get all the content. But I want to get the node at the cursor position or where they're typing. 

I can see it's possible to get the [top-level node](https://github.com/orgs/Milkdown/discussions/61), or just the prose nodes, but how can you work out the existing node from that?

**Comment by shenzhongkang** [comment]
```ts
const resolvedPos = view.state.doc.resolve(pos);
// marks
const marks = resolvedPos.marks();
// node
const node = resolvedPos.node();
```
